components/Navigation/index.js

```diff
@@ -14,7 +14,11 @@ export default function Navigation({ onCreateClick }) {
       <StyledNavButtonBox>
         {router.pathname === "/joined" ? (
           <StyledSideLink href="/">
-            <TbArrowBackUp size={theme.button.xs} color={theme.textColor} />
+            <TbArrowBackUp
+              size={theme.button.xs}
+              color={theme.textColor}
+              style={{ transform: "scaleX(-1)" }}
```

Comment:
no inline styling :)

Reply:
I had to use it here because of the react-icon behavior. Styled components doesn't work.